src/components/tools/text-tool.tsx

```diff
@@ -302,6 +302,12 @@ export default class TextToolComponent extends BaseComponent<IToolTileProps, ISt
     return this.props.model.content as TextContentModelType;
   }
 
+  private handleOnBlur = () => {
```

Comment:
A react convention which we (mostly/sometimes) follow is that `onFoo` is the name of the prop and `handleFoo` is the name of the function.
```suggestion
  private handleBlur = () => {
```

Reply:
Done!